OpenRA.Mods.Common/TraitsInterfaces.cs

```diff
@@ -437,6 +437,16 @@ public interface IMove
 		bool CanEnterTargetNow(Actor self, Target target);
 	}
 
+	public interface IPositionOffset
```

Comment:
Can we please call this `IAircaftCenterPositionOffset`, since this only applies to aircraft.

Reply:
I deliberately gave it a more generic name since the concept doesn't have to be exclusive to aircraft. It includes horizontal offsets as well, which could be useful for extending the carryall logic to land units.

Reply:
My issue here is that right now it *is* exclusive to aircraft, so it is misleading to name it otherwise.
We can rename it to `ICenterPositionOffset` in the future if `Mobile`or other traits add support.

Reply:
Changed the name.